publish/publish.proj

```diff
@@ -106,8 +106,8 @@
     
     <Message Text="Pushing CoreHost packages to $(NuGetFeedUrl)" />
     <PropertyGroup>
-      <NuGetPushCommand>$(NuGetToolPath) push -Source $(NuGetFeedUrl) -ApiKey $(NuGetApiKey) -Timeout $(NuGetPushTimeoutSeconds)</NuGetPushCommand>
```

Comment:
I'm going to test out setting buildnumberminor to 99 for non-official builds.  I'd expect that to prevent this type of failure until we get out of the "preview2-*" versioning in test projects